src/components/Header/Header.jsx

```diff
@@ -1,4 +1,9 @@
 import styles from './Header.module.css';
+import { CustomInput } from "../CustomInput/CustomInput";
+
+import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
```

Comment:
import them all on the same line, this will make it easier to import other ones in the future